log4j-core/src/test/java/org/apache/logging/log4j/core/HostNameTest.java

```diff
@@ -16,21 +16,24 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
 import java.util.List;
 
 import org.apache.logging.log4j.core.appender.RollingFileAppender;
 import org.apache.logging.log4j.core.util.NetUtils;
 import org.apache.logging.log4j.junit.LoggerContextRule;
 import org.apache.logging.log4j.test.appender.ListAppender;
 import org.apache.logging.log4j.util.Strings;
+
```

Comment:
We don't require whitespace here. We don't disallow it either, but not sure what the point is.